### DISCUSSION TITLE
Salvar dados com a planilha aberta é possível ?

### DISCUSSION THREAD

**Discussion by ArrudaFernando** [exceljs/exceljs#2207] [Q&A]
Preciso salvar os dados na planilha mesmo quando ela estiver aberta, porém só estou conseguindo com ela fechada. É possóivel preencher as células com a planilha aberta ? Caso sim, como ?

**Comment by renatocfrancisco** [comment]
É, acho que não é possível devido à permissão de arquivo. Não pode estar aberto em dois lugares. **Acho.**
